test/e2e/lib/components/site-editor-component.js

```diff
@@ -127,4 +127,33 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 			By.css( '.edit-site-header-toolbar__list-view-toggle' )
 		);
 	}
+
+	async removeBlock( blockID ) {
+		await this.runInCanvas( async () => {
+			const blockLocator = By.css( `.wp-block[id="${ blockID }"]` );
+			await driverHelper.waitUntilElementLocatedAndVisible(
+				this.driver,
+				blockLocator,
+				this.explicitWaitMS / 5
+			);
+			await this.driver.findElement( blockLocator ).click();
+		} );
+
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css( '.block-editor-block-settings-menu' )
+		);
+		await driverHelper.waitUntilElementLocatedAndVisible(
+			this.driver,
+			By.css( '.components-menu-group' ),
+			this.explicitWaitMS / 5
+		);
+		await this.driver.sleep( 1000 );
```

Comment:
Why the extra sleep here?

Reply:
This is a copy-paste from `GutenbergEditorComponent#removeBlock`, but adapted to the site editor. The `sleep` is not documented there either, so I'm not quite sure why it's required.

Reply:
Ah, OK. Anyway, a good approach is to verify if those mysterious `sleeps` are actually necessary whenever bumped into one, especially when copypasting from somewhere else. This way we're not adding any unnecessary time to our E2Es.